.github/build.in.yml

```diff
@@ -126,7 +126,7 @@ jobs:
       - name: get cache
         id: get
         run: |
-          rm -rf .lake/build/lib/Mathlib/
+          rm -rf .lake/build/lib/lean/Mathlib .lake/build/lib/Mathlib/
```

Comment:
Let's split this into two commands, so we can leave a comment saying that one is for cleaning up pre-4.18.0-rc1 oleans.

Reply:
Split!